src/README.md

```diff
@@ -424,6 +424,23 @@ This should only be performed if it is actually sure that the operation has
 not failed. A lot of Node.js’s source code does **not** follow this rule, and
 can be brought to crash through this.
 
+In particular, it is often not safe to assume that an operation does not throw
+an exception, even if it seems like it would not do that.
+The most common reasons for this are:
+
+* Calls to functions like `object->Get(...)` or `object->Set(...)` may fail on
+  most objects, if the `Object.prototype` object has been modified from userland
+  code that added getters or setters.
+* Calls that call *any* JavaScript code, including JavaScript code that is
```

Comment:
Making this suggestion just because "calls that call" sounds a little off IMO.

```suggestion
* Calls that invoke *any* JavaScript code, including JavaScript code that is
```